src/TurboLinks.Net/TurboLinks.Net.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Add support for Turbolinks</Description>
+    <VersionPrefix>1.1.1</VersionPrefix>
+    <Authors>Tommy Parnell</Authors>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>TurboLinks.Net</AssemblyName>
+    <PackageId>TurboLinks.Net</PackageId>
+    <PackageTags>TurboLinks Middlewear</PackageTags>
+    <PackageProjectUrl>https://github.com/tparnell8/TurboLinks.Net</PackageProjectUrl>
+    <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
When you target both netstandard1.6, and 2.0. Please make this item group conditional on targetframework 1.6, and make a new ItemGroup with updated versions of these packages for netstandard2.0